### DISCUSSION TITLE
Plusieurs WIT un seul avec synapps ??

### DISCUSSION THREAD

**Discussion by Baptiste-ALL** [witsa/synapps#753] [Q&A]
Bonjour j'ai sur mon site 1 dizaine de WIT ancienne génération (e@sy) avec des synoptiques déjà fait.  j'ai un WIT (REDY) sur lequel je travail pour une supervision avec synapps. sur l'ancienne supervision nous pouvions a partir d'une page accéder a tout les WIT sans rouvrir de nouvelle fenêtre et donc accéder au pilotage. et t'il possible d'accéder avec synapse au synoptique des autres WIT en mode navigateur ou autre?  si non faut t'il les changé par des REDY et pouvoir ouvrir un synapps dans un synapps???
Merci

**Comment by WITNicolas** [comment]
Bonjour,
Il suffit d'utiliser un acteur de type iFRAME.
Dans cet acteur on peut renseigner une URL de connexion vers 1 easy en mettant un chemin complet
_ex : http://192.168.1.10/easy/syn/Syn_1_

ça serait sensiblement le même principe pour une SYNAPP dans une SYNAPP (avec des contraintes de sécurité quand même)

**Comment by Baptiste-ALL** [comment]
Merci pour l'information j'ai donc crée une iframe avec la synoptique du wit sur le logiciel cela marche bien mais une fois chargé dans le REDY pas d'affichage et pas de défaut je me suis connecté en admin pareil ... je me connect en externe et le " WSID0485320273 " a donc changé avais vous une solution? je sais que sur la page dont je vous par le lors de mon premier message il y avait un script de ce style la :
 seesVarWrite("SessURLAmadeusS";"http://***.248.***.***:1**1/")
     SessVarWrite("SessWLog";"?wlog="&&StrField(Value(SessUserPath&&".Identity");1;"_")&&":"&&Value(SessUserPath&&".PSw"))
D'avance merci

**Comment by Baptiste-ALL** [comment]
Bonjour en rajoutant la case chargement de contenu cela me ferme la synapps et m'ouvre directement la page de connexion au WIT.

**Comment by WITNicolas** [comment]
Bonjour.
Dans l'URL de connexion vous pouvez mettre : 
http://192.168.1.10/easy/syn/Syn_1?wlog=admin:.

par contre c'est une pratique qui laisse potentiellement une faille de sécurité.
coté SYNAPP, il n'est pas possible de récupérer le mot de passe utilisateur pour le faire transiter. (question de sécurité)

Ce qui était très (trop) permissif avec le SYNOPTIQUE et devenu plus contraignant avec STUDIO

**Comment by Baptiste-ALL** [comment]
Bonjour, 
Je suis d'accord avec vous coté sécurité cela n'est pas génial. Peut être travaillé sur un système avec studio qui serais sécurisé car je pense que plein de personne veulent pouvoir gérer un parc de Wit sans forcement changé de synapps avec mot de passe etc...

Merci à vous et belle journée
